### ISSUE TITLE
Latest chart version is outdated (1.6.0)

### ISSUE THREAD

**Issue by bkleef** [jfelten/gitea-helm-chart#32]
Please push current (1.9.3) chart version (from master branch) to `https://jfelten.github.io/helm-charts/charts`.

Steps to reproduce:

```sh
helm repo add jfelten https://jfelten.github.io/helm-charts/charts
helm repo update
helm search gitea
```

```sh
NAME         	CHART VERSION	APP VERSION	DESCRIPTION
jfelten/gitea	1.6.0        	           	Git with a cup of tea
```

**Comment by jfelten** [comment]
ok will spend some time on this

**Comment by robvadai** [comment]
Yes please update to the latest version

**Comment by luqasn** [comment]
Yeah, that would be great! For now, I have forked it and created a helm repo with only 1.9.3 in it: https://luqasn.github.io/gitea-helm-chart but would love to go back to "upstream" :)

**Comment by wagnst** [comment]
> Yeah, that would be great! For now, I have forked it and created a helm repo with only 1.9.3 in it: https://luqasn.github.io/gitea-helm-chart but would love to go back to "upstream" :)

Then do a PR with the changes....

**Comment by luqasn** [comment]
> > Yeah, that would be great! For now, I have forked it and created a helm repo with only 1.9.3 in it: https://luqasn.github.io/gitea-helm-chart but would love to go back to "upstream" :)
> 
> Then do a PR with the changes....

Sure, I could create a PR on the jfelten/helm-charts repo, but looks like jfelten themselves would need to merge that. I guess it would be better to add something like this to THIS repo: https://github.com/helm/chart-releaser-action

What do you think?

**Comment by wagnst** [comment]
> > > Yeah, that would be great! For now, I have forked it and created a helm repo with only 1.9.3 in it: https://luqasn.github.io/gitea-helm-chart but would love to go back to "upstream" :)
> > 
> > 
> > Then do a PR with the changes....
> 
> Sure, I could create a PR on the jfelten/helm-charts repo, but looks like jfelten themselves would need to merge that. I guess it would be better to add something like this to THIS repo: https://github.com/helm/chart-releaser-action
> 
> What do you think?

I can merge as well. Yes that might be a place to move the chart, but that's something @jfelten might need to think about...

**Comment by luqasn** [comment]
https://github.com/jfelten/helm-charts/pull/1

**Comment by luqasn** [comment]
After there was no activity on the PR for a month, it might be easier to go the GitHub actions route. If interested, check out this PR: https://github.com/luqasn/gitea-helm-chart/pull/1

If there is interest in this, I can rebase and make a PR here. One of the benefits is that this also test-reploys to a kubernetes cluster (see [here](https://github.com/luqasn/gitea-helm-chart/commit/f7d755adb55160b792b0c6caaa21e19898359fe9/checks?check_suite_id=395699267) for an example run)

**Comment by huats** [comment]
I think we can close this since luqasn#1 has been merged and that the chart provides version 1.9.3

**Comment by jfelten** [comment]
closing
